### DISCUSSION TITLE
Support @par and other Doxygen commands

### DISCUSSION THREAD

**Discussion by zyphlar** [JakubAndrysek/MkDoxy#88] [General]
Got this error:

```console
  File "C:\Users\me\AppData\Local\Packages\PythonSoftwareFoundation.Python.3.10_qbz5n2kfra8p0\LocalCache\local-packages\Python310\site-packages\mkdoxy\xml_parser.py", line 247, in paras
    ret.extend((Br(), MdBold([Text(SIMPLE_SECTIONS[kind])])))
KeyError: 'par'
```

As a temporary workaround I added this line to SIMPLE_SECTIONS in xml_parser.py:

`"par": "\r\n",`

Support for all/most Doxygen commands (or at least an easy workaround or warning for unknown commands) would be great:
https://www.doxygen.nl/manual/commands.html

**Comment by JakubAndrysek** [comment]
Connected branch: https://github.com/JakubAndrysek/MkDoxy/tree/add-more-doxygen-commands

**Comment by JakubAndrysek** [comment]
Recently added:
- /par

**Comment by JakubAndrysek** [comment]
TODO
Research similar tools:
https://breathe.readthedocs.io/en/latest/codeguide.html
